examples/typescript-reactflux/externs/browserify.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
@arthurvr No, these extern files are for the closure compiler. Otherwise you can't compile with [ADVANCED_OPTIMIZATTION](https://developers.google.com/closure/compiler/docs/api-tutorial3). I couldn't find any extern definitions for `browserify` (happens a lot), there are no npm modules.